projects/ui-policy-studio-angular/src/lib/components/flow-form-dialog/flow-llm-form-dialog/gio-ps-flow-llm-form-dialog.component.ts

```diff
@@ -27,20 +27,22 @@ import {
   GioElAssistantComponent,
   GioFormSlideToggleModule,
   GioFormTagsInputModule,
+  GioFormAutocompleteInputModule,
   GioIconsModule,
 } from '@gravitee/ui-particles-angular';
 
 import { GioPolicyStudioFlowFormDialogResult } from '../gio-ps-flow-form-dialog-result.model';
-import { ConditionSelector, HttpMethod, HttpMethods, HttpSelector } from '../../../models';
+import { ConditionSelector, HttpMethod, HttpSelector } from '../../../models';
 import { FlowVM } from '../../../policy-studio/gio-policy-studio.model';
 
 export type GioPolicyStudioFlowLlmFormDialogData = {
   flow?: FlowVM;
 };
 
-type HttpMethodVM = HttpMethod | 'ALL';
+type HttpMethodVM = HttpMethod;
 
-const METHODS_AUTOCOMPLETE: HttpMethodVM[] = ['ALL', ...HttpMethods];
+const METHODS_AUTOCOMPLETE: HttpMethodVM[] = ['POST', 'GET'];
+const LLM_PATHS: string[] = ['/chat/completions', '/embeddings', '/models'];
```

Comment:
I recommend creating a typescript tuple with the paths (instead of `string[]`). That way, typescript can help if a new one is added (for example, completing the switch below).